coordinator/src/main/scala/filodb.coordinator/queryplanner/LongTimeRangePlanner.scala

```diff
@@ -34,8 +34,8 @@ class LongTimeRangePlanner(rawClusterPlanner: QueryPlanner,
     logicalPlan match {
       case p: PeriodicSeriesPlan =>
         val earliestRawTime = earliestRawTimestampFn
-        lazy val offsetMillis = LogicalPlanUtils.getOffsetMillis(logicalPlan)
-        lazy val lookbackMs = LogicalPlanUtils.getLookBackMillis(logicalPlan)
+        lazy val offsetMillis = LogicalPlanUtils.getOffsetMillis(logicalPlan).max
+        lazy val lookbackMs = LogicalPlanUtils.getLookBackMillis(logicalPlan).max
         lazy val startWithOffsetMs = p.startMs - offsetMillis
         lazy val endWithOffsetMs = p.endMs - offsetMillis
```

Comment:
endWithOffsetMs should use min offset, not max.

Reply:
Can you make sure that in your PR, maxOffset/maxLookback is used with startTime and minOffset/minLookback is used with endTime ? I dont see it consistently

Reply:
Yes, you are right. I have made the changes. Thanks!